src/test/e2e/tests/specs.test.ts

```diff
@@ -4,24 +4,17 @@ import { AdminDashboardPage } from '../PageObjects/AdminDashboard.page';
 import { SystemAdminDashboardPage } from '../PageObjects/SystemAdminDashboard.page';
 import { AlphabeticalSearchPage } from '../PageObjects/AlphabeticalSearch.page';
 import { CaseEventGlossaryPage } from '../PageObjects/CaseEventGlossary.page';
-import { CaseNameSearchPage } from '../PageObjects/CaseNameSearch.page';
```

Comment:
These files for these pages as well can also be removed too :)

Reply:
I will remove them :)